### ISSUE TITLE
Make mobile search form accessible: 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#269]
_Epic: #265_

> The search should be implemented the same way [[ as #266, #267, #268 ]], with a keyboard trap that keeps focus on the search until the user selects the close button. In this case, since there is no close button the focus should return back to the search button icon after the check box option.

![screen shot 2018-04-27 at 3 06 40 pm](https://user-images.githubusercontent.com/263285/39382857-a2e76ab4-4a2c-11e8-919b-049b2df2bc98.png)

WCAG 2.0 Compliance: ???

Traceability: [Microassist Report 1 > WCAG Compliance > Name, Role, Value](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.t8cmbkue4bcm)

**Comment by paregorios** [comment]
Please merge to master and deploy to production.
